appveyor.yml

```diff
@@ -23,7 +23,8 @@ environment:
   USE_RTOOLS: true
   NOT_CRAN: true
   KEEP_VIGNETTES: false
-  R_BUILD_ARGS: --no-build-vignettes --no-manual"
```

Comment:
This line difference is only the quote at the end. I think it was there accidentally before.